README.md

```diff
@@ -33,6 +50,7 @@ Dynamic Extensions is also known to work on:
 * Alfresco Enterprise 4.0
 * Alfresco Enterprise 5.0
 * Alfresco Enterprise 5.1
+* Alfresco Enterprise 5.2
```

Comment:
Tests are run for Alfresco Community 5.2, so maybe that version should be added to _Dynamic Extensions is developed and tested against_